src/state/is-within-visible-bounds-of-droppable.js

```diff
@@ -8,10 +9,32 @@ import type {
 } from '../types';
 
 export const isPointWithin = (droppable: DroppableDimension) => {
-  const { top, right, bottom, left } = droppable.page.withMargin;
+  const droppableRect = droppable.page.withMarginAndPadding;
+  const { container } = droppable;
 
-  const isWithinHorizontal = isWithin(left, right);
-  const isWithinVertical = isWithin(top, bottom);
+  // Calculate the mid-drag scroll ∆ of the scroll container
+  const containerScrollDiff = subtract(container.scroll.current, container.scroll.initial);
+
+  // Calculate the droppable's bounds, accounting for the container's scroll
+  const droppableBounds = {
```

Comment:
we seem to do this type of operation a lot: `Spaceing` add or subtract a `position`. Perhaps it is worth creating a utlity for this in a `spacing.js` file